@css/main_style.css

```diff
@@ -28,6 +28,50 @@
 
 ************************************************************************************************************************/
 
+/* ******************************** *
+ * ROOT element                     *
+ * ------------                     *
+ * CSS Global variables declaration *
+ * ******************************** */
+
+:root {
+  /* Global page values */
+  --global-page-width: 375px;
+  --global-page-height: 750px;
+
+  /* Separator values */
+  --separator-width: var(--global-page-width);
+  --separator-height: 114px;
+
+  /* Section 1 */
+  --section_1-width: 375px;
+  --section_1-height: 588px;
+  --section_1-padding-left: 40px;
+  --section_1-padding-right: 40px;
+
+  /* Section 2 */
+  --section_2-width: 375px;
+  --section_2-height: 588px;
+  --section_2-padding-left: 40px;
+  --section_2-padding-right: 40px;
+
+  /* Workcard */
+  --workcard-width: 327px;
+  --workcard-height: 596px;
+  --workcard-padding-left: 24px;
+  --workcard-padding-right: 24px;
+
+  /* Workcard image */
```

Comment:
- [ ] Please make sure that you have fulfilled all the requirements, as your previous code reviewer asked for changes related to converting the absolute value to relative value by using %age to make your design responsive but it seems like you didn't alter the changes asked by previous code reviewer. Kindly have a look at the below requirement image of the project.

### Requirement
![image](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/66476329/bfe1dd10-aa03-49ae-ad4f-ab7d4ad781f2)